src/test/java/NumbersTest.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NumbersTest {
```

Comment:
Poniższe komentarze należy zastosować we wszystkich miejscach, w których pojawia się dany problem.